### PR TITLE
#18 Add support for SHACL shapes

### PR DESCRIPTION
**Pull request by yashkhatri**
Refactored SHACL related code from Lyo Validation and moved to a separate project under lyo core repository.

Change-Id: If963bf2d3bc1ba2e097c0d93a911bc067cab9546
Signed-off-by: Yash Khatri <yash.khatri@scania.com>

### CODE REVIEW

**Comment by jadelkhoury** [comment]
@yashkhatri Is there more than a pure move of code here?
too many files to review. If there are any code changes, maybe you can point them out, so I can focus on them?

otherwise, I think @berezovskyi is better to review how the code is structured within core

File: org.eclipse.lyo.oslc4j.build/pom.xml [hunk 1]
```diff
@@ -41,6 +41,7 @@
     <module>../org.eclipse.lyo.oslc4j.wink</module>
     <module>../org.eclipse.lyo.oslc4j.registry</module>
     <module>../org.eclipse.lyo.repackaging.jersey</module>
+    <module>../org.eclipse.lyo.shacl</module>
   </modules>
```
Comment (yashkhatri) [inline org.eclipse.lyo.oslc4j.build/pom.xml]: @jadelkhoury This line is new. Otherwise it is just a pure code move from Lyo Validation repository.

**Comment by jadelkhoury** [comment]
looks good to me.

**Comment by berezovskyi** [comment]
This one is still not merged. Do you need any input from me?

**Comment by yashkhatri** [comment]
@berezovskyi Just double check if we need any CQ and we are good to merge. (This is same code from Lyo Validation. So, if you have already reviewed there, no need to review again. )

**Comment by jadelkhoury** [comment]
@berezovskyi and I wanted you to have the final word on the naming of the project.

**Comment by berezovskyi** [comment]
not the same code. you forgot to mention that you changed the package, so all previously dependent code will break.

**Comment by berezovskyi** [comment]
@jadelkhoury @jamsden we'll need your opinion about it here

**Comment by jadelkhoury** [comment]
as discussed with @berezovskyi today, it is ok if previous code breaks, since we don't have that many users yet.
Better to use hte more appropriate package naming.

**Pull request merged**